content-src/styles/_variables.scss

```diff
@@ -75,6 +78,8 @@ $about-welcome-extra-links: #676F7E;
 $firefox-wordmark-default-color: #363959;
 $firefox-wordmark-darktheme-color: $white;
 $trailhead-purple: #2B2156;
+$trailhead-blue-60: #0250BB;
+$trailhead-start-hover-color: #B833E1;
```

Comment:
This looks a bit jarring to me and inconsistent with the blue continue button, @aaronrbenson what do you think?